app/src/js/components/SortableTable/SortableTable.js

```diff
@@ -83,19 +98,31 @@ const SortableTable = ({
     }),
     []
   );
-  const [fitColumn, setFitColumn] = useState({});
-  const [leftScrollButtonVisibility, setLeftScrollButtonVisibility] = useState({ display: 'none', opacity: 0 });
-  const [rightScrollButtonVisibility, setRightScrollButtonVisibility] = useState({ display: 'none', opacity: 0 });
-  let sortByState;
+
```

Comment:
It seems like there are a lot of changes here unrelated to the React Router update. Can you give a quick overview of the objective here?

Reply:
It's still there. I just put state together and moved the let with the other variables. Some of our code is disorganized and I needed to clean it up to understand where everything is. Sorry for any confusion.

<img width="839" alt="Screenshot 2025-02-18 at 4 30 52 PM" src="https://github.com/user-attachments/assets/1197d9a7-55d3-4124-aba7-e9b034bae7c4" />